ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/individual-journey/individual-journey.spec.ts

```diff
@@ -171,15 +171,6 @@ describe('IndividualJourney', () => {
     expectStep(redirected).toBe(Steps[Steps.GotoVideoApp]);
   });
 
-  it('should goto video app if trying to enter a finished journey', () => {
```

Comment:
is this not true for both individual and representative? In which case, maybe it can be put into a test class for the base class?